models/exchange/kucoin/api.py

```diff
@@ -888,11 +888,16 @@ def getHistoricalData(
                 ["date", "market", "granularity", "low", "high", "open", "close", "volume"]
             ]
 
-            df["low"] = pd.to_numeric(df["low"])
-            df["high"] = pd.to_numeric(df["high"])
-            df["open"] = pd.to_numeric(df["open"])
-            df["close"] = pd.to_numeric(df["close"])
-            df["volume"] = pd.to_numeric(df["volume"])
+            #df["low"] = pd.to_numeric(df["low"])
+            #df["high"] = pd.to_numeric(df["high"])
+            #df["open"] = pd.to_numeric(df["open"])
+            #df["close"] = pd.to_numeric(df["close"])
+            #df["volume"] = pd.to_numeric(df["volume"])
```

Comment:
there's not much reason to keep dead code in the comments here. I'd rather remove the old code.